File: 30 Days of Code/Ruby/Day25_RunningTimeComplexity.rb
n = gets.to_i

for j in 0...n do
    num = gets.to_i
    flag = true
    
    for j in 2..Math.sqrt(num) do
        if num.remainder(j) == 0
            flag = false
            break
        end
    end
    
    if num <=1
        flag = false
    end
    
    if flag
        puts "Prime"
    else puts "Not prime"
    end
end
    

File: 30 Days of Code/Python/Day25_RunningTimeComplexity.py
import math as m
for i in range(int(input())):
    num  = int(input())
    
    if num > 1:
        flag = True
        for j in range (2, int (m.sqrt(num))+1):
            if num%j == 0:
                flag = False
                break
        if flag :
            print("Prime")
        else:
            print("Not prime")
    else:
        print("Not prime")
                